test/ex_webrtc/peer_connection_test.exs

```diff
@@ -36,79 +36,83 @@ defmodule ExWebRTC.PeerConnectionTest do
                  fingerprint: {:sha256, @fingerprint}
                )
 
-  test "negotiation needed" do
```

Comment:
I had to separate those cases as it's hard to determine which one exactly was failing